CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 5.6.1
```

Comment:
Hmmm this should be 5.6.2 🤔

Reply:
I believe this should fix this: https://github.com/RevenueCat/purchases-android/pull/624